pkg/internal/checkup/executor/executor.go

```diff
@@ -32,6 +32,7 @@ import (
 	"kubevirt.io/client-go/kubecli"
 
 	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/console"
+	trex_client "github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/trex-client"
```

Comment:
How about changing the package name to `trex` (same for the directory), and then we will have `trex.Client`.

Reply:
changed package name to trex